plugins/tech-insights-backend/api-report.md

```diff
@@ -24,6 +24,14 @@ export const buildTechInsightsContext: <
   options: TechInsightsOptions<CheckType, CheckResultType>,
 ) => Promise<TechInsightsContext<CheckType, CheckResultType>>;
 
+// Warning: (ae-forgotten-export) The symbol "Options" needs to be exported by the entry point index.d.ts
```

Comment:
Can we fix up some of these warnings and export the types or inline them instead.

Reply:
I ended up with quite a different change but I've updated the api report (no warnings)